pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go

```diff
@@ -2424,7 +2424,7 @@ var _ = Describe("Validating VM Admitter", func() {
 		resp := admitVm(vmsAdmitter, vm)
 		Expect(resp.Allowed).To(BeTrue())
 		Expect(resp.Result).To(BeNil())
-		Expect(resp.Warnings).To(HaveLen(1))
+		Expect(resp.Warnings).To(HaveLen(2))
```

Comment:
Let's assert content of the warnings

Reply:
added expectations for the content of the warnings